x-pack/plugins/security_solution/public/timelines/components/modal/actions/save_timeline_button.tsx

```diff
@@ -22,9 +22,18 @@ export interface SaveTimelineButtonProps {
    */
   timelineId: string;
   /**
-   * Ability to customize the text of the button
+   * Customize the button
    */
-  buttonText?: string;
+  options?: {
```

Comment:
`options` wasn't clear to me what it is at first glance and i'm not sure this is needed? seems reasonable to have props like `buttonText` and `color` for a component called `saveTimelineButton`

Reply:
that is a good point and I had this at first but I could see the number of props growing so decided to nest them under a single one just in case. I'm usually the first one to say don't write code for a potential future and I think that applies here. I'll revert back to 2 props `buttonText` and `buttonColor`.
Thank you!